### ISSUE TITLE
Frontend

### ISSUE THREAD

**Issue by MariusBluem** [nextcloud/user_external#28]
SAML/SSO and LDAP are configurable via webinterface **and** config.php. We should make this possible for backends like IMAP and FTP too ...

@nextcloud/user_external

**Comment by violoncelloCH** [comment]
started some work in #46

**Comment by Mannshoch** [comment]
any news on this?

**Comment by Mannshoch** [comment]
Mockup
![grafik](https://user-images.githubusercontent.com/7795891/167830034-7782459c-a34e-40e8-8bef-c4c85d884439.png)
It addresses following Issues:
https://github.com/nextcloud/user_external/issues/127
https://github.com/nextcloud/user_external/issues/12

**Comment by ThePatzen** [comment]
Hi!

Any news on this topic? Would be a huge improved because of some managed nextcloud-instances do not support access to config.php....

Best Regards!

**Comment by dprandzioch** [comment]
Yeah have the same problem here, Hetzner managed Nextcloud does not support accessing config.php, is there any progress or workaround to make this work?
